src/app/templates/app/convert.html

```diff
@@ -138,41 +133,52 @@ <h4 id="doclabel">Upload SPDX Document using the button or by dragging and dropp
   $("#convertpage").addClass('linkactive');
 
   $("#from_format").change(function() {
-    $("#tagToRdfFormat").prop("disabled",'disabled');
-    $(".tagToRdfOptions").css("color",'grey');
     if ($(this).data('options') === undefined) {
       /*Taking an array of all options-2 and kind of embedding it on the select1*/
       $(this).data('options', $('#to_format option').clone());
     }
     var id = $(this).val();
-    var options = $(this).data('options').filter('[data-value=' + id + ']');
+	var id2 = $('#to_format').val();
+    var options = $(this).data('options').filter('[data-value!=' + id + ']');
     $('#to_format').html(options);
     if (id!="0"){
       $('#doclabel').text("Upload SPDX "+ id +" Document using the button or by dragging and dropping onto the dashed region");
     }
     else{
       $('#doclabel').text("Upload SPDX Document using the button or by dragging and dropping onto the dashed region");
     }
-    var id2 = $('#to_format').val();
-    if (id2=="Spreadsheet"){
+	if (id2 == id) {
+	    $('#to_format').val("0");
+    } else {
+		$('#to_format').val(id2);
+	}
+    if (id2=="XLS"){
       $('#convertdoclabel').text(".xls");
       $('#cfileformat').val(".xls");
     }
-    else if (id2=="RDF"){
-      if(id=="Tag"){
-        $("#tagToRdfFormat").prop("disabled", false);
-        $(".tagToRdfOptions").css("color",'black');
-      }
-      $('#convertdoclabel').text(".rdf");
-      $('#cfileformat').val(".rdf");
+	else if (id2=="XLSX"){
+      $('#convertdoclabel').text(".xlsx");
+      $('#cfileformat').val(".xlsx");
     }
-    else if (id2=="Tag"){
+    else if (id2=="RDFXML"){
+      $('#convertdoclabel').text(".rdf.xml");
+      $('#cfileformat').val(".rdf.xml");
+    }
+    else if (id2=="TAG"){
       $('#convertdoclabel').text(".spdx");
       $('#cfileformat').val(".spdx");
     }
-    else if (id2=="HTML"){
-      $('#convertdoclabel').text(".html");
-      $('#cfileformat').val(".html");
+    else if (id2=="JSON"){
+      $('#convertdoclabel').text(".json");
+      $('#cfileformat').val(".json");
+    }
+	else if (id2=="YAML"){
+      $('#convertdoclabel').text(".yaml");
+      $('#cfileformat').val(".yaml");
+    }
+	else if (id2=="XML"){
+      $('#convertdoclabel').text(".xml");
+      $('#cfileformat').val(".xml");
```

Comment:
These if else can be combined by using html data property, will commit the changes.

Reply:
Fixed in my commit.